### DISCUSSION TITLE
Local file read

### DISCUSSION THREAD

**Discussion by scollis** [blaylockbk/Herbie#157] [Ideas]
Herbie has some useful accessors such as the get_point. I want to grab subset and save netcdf via xarray in Herbie and then reload with the Herbie xarray accessors still available.

**Comment by blaylockbk** [comment]
That is a great idea! I'm so glad you are sharing your thoughts while using Herbie. You are giving me a lot of great ideas and areas of improvement.

**Comment by scollis** [comment]
No, Thank you :) 
This codebase fills a nice need.
